principles/security-principles.md

```diff
@@ -2,8 +2,31 @@
 layout: default
 title:  "Security Principles"
 ---
+# Penetration Testing Process
+When a project is close to completion delivery managers will need to liase with the security team to arrange a penetration test. 
 
-# Security
+DM's will need to fill out the scoping document which can be found at the following links:
+1. [Web application assessments](https://github.com/SkillsFundingAgency/das-technical-guidance/blob/security-principles-update/principles/scoping_documents/Web_App_Scoping_Questionnaire%20(3).docx)
+2. [API assessments](https://github.com/SkillsFundingAgency/das-technical-guidance/blob/security-principles-update/principles/scoping_documents/API_Scoping_Questionnaire%20(1).docx)
 
-Placeholder
+Should the team require any clarification with regard to the content of the scoping form or need to confirm scope they can contact the security team for assisstance.
+
+Once the scoping forms have been returned the security team may arrange for a walkthrough of the product to clarify scope.
+
+When testing is complete the the delivery manager will receive an encrypted penetration testing report, which can then be distributed to their team as they see fit.
```

Comment:
It would be useful to include the Architect on the project in this report. Or at least copy it to Roy and myself so that we can ensure the teams are taking the appropriate steps on the report.

Reply:
Sure, I will ensure that both you and Roy and CC'd in future for each report that is delivered